swimster-ui/src/components/CreatePoolSteps/CreatePoolSteps.jsx

```diff
@@ -0,0 +1,145 @@
+}
+
+const CreatePoolSteps = ({ step, nextStep, prevStep, register, setValue, resetField }) => {
+    const [amenitiesChosen, setAmenitiesChosen] = useState([])
+
+    // allows selecting and deselecting of amenities
+    const handleSelected = (amenity) => {
+        if (getSelected(amenity)) {
+            const newSeletedAmenities = amenitiesChosen.filter(amenityTitle => amenityTitle !== amenity.title)
+            return setAmenitiesChosen(newSeletedAmenities)
+        }
+        setAmenitiesChosen([...amenitiesChosen, amenity.title])
+    }
+
+    // checks if amenity is selected for UI change
+    const getSelected = amenity => amenitiesChosen.includes(amenity.title)
+
+    // resets these fields - solves autocomplete fields bug
+    useEffect(() => {
+        resetField("poolLength")
+        resetField("poolWidth")
+        resetField("poolDepth")
+    }, [step])
+
+    switch (step) {
+        case 1:
+
+            return (
+                <>
+                    <h1 className={style.formCreatePoolHeader}>Get Started with the Basics.</h1>
+                    <input
+                        className={`${style.inputElement} ${style.fullInput}`}
+                        type="text"
+                        placeholder="Title"
+                        {...register("title")}
+                    />
+                    <input
+                        className={`${style.inputElement} ${style.fullInput}`}
+                        type="text"
+                        placeholder="Address"
+                        {...register("address")}
+                    />
+                    <textarea
+                        className={`${style.inputElement} ${style.fullInput}`}
+                        rows={6}
+                        placeholder="A little description..."
+                        {...register("description")}
+                    />
+                    <input
+                        className={style.inputElement}
+                        type="number"
+                        placeholder="Price"
+                        {...register("price")}
+                    />
+                    <input
+                        className={style.inputElement}
+                        type="number"
+                        placeholder="Guests"
+                        {...register("totalGuests")}
+                    />
+                    <select
+                        className={style.inputElement}
+                        type="text"
+                        placeholder="Pool Type"
+                        {...register("poolType")}
+                    >
+                        <option value="Outdoors">Outdoors</option>
+                        <option value="Indoors">Indoors</option>
+                    </select>
+                    <input
+                        className={`${style.inputElement} ${style.fullInput}`}
+                        type="text"
+                        placeholder="Image URL"
```

Comment:
have you and @erihuan2 talked about the backend for images?

Reply:
Not yet, but I'm currently working on that and will reach out to him soon